dlt/normalize/worker.py

```diff
@@ -111,7 +111,7 @@ def _get_items_normalizer(
             if item_format == "file":
```

Comment:
I think `item_format_from_file_extension` should return `model` item format, not `file` for models. here this is not relevant but IMO may be in other places

Reply:
It does already doesn't it? Or am I misunderstanding your comment.